src/vm/hints/secp/secp_utils.rs

```diff
@@ -13,8 +18,7 @@ pub fn split(integer: &BigInt) -> Result<[BigInt; 3], VirtualMachineError> {
     if integer.is_negative() {
         return Err(VirtualMachineError::SecpSplitNegative(integer.clone()));
     }
-    let base = bigint!(1) << 86_usize;
-    let base_max = base - bigint!(1);
+    let base_max = &*BASE_86 - bigint!(1);
```

Comment:
This can also be `lazy_static!`.

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/329/commits/ea093c1dac6342877c95a5c6dc001593608f795f